### DISCUSSION TITLE
Add more information in reports

### DISCUSSION THREAD

**Discussion by plearn2023** [vmware/powershell-module-for-vmware-cloud-foundation-reporting#140] [Community Help]
Hello, I would like to suggest adding a feature that provides more information about critical VMs in SDDC such as Center (IP, Subnet Mask, Port group, vLAN), and similarly for NSX manager. It would be worth if ESXi network details including MTU, vSAN Network, vmkx, and other relevant details could also be included. 
A recent scenario where 6 out of 8 vSAN nodes rebooted resulted in us losing access to vCenter. We had no information available because our CMDB was also down. GSS requested to restore vCenter from backup, and we had to rebuild the ESXi host. This proved to be almost impossible because we were missing so many details. We believe that including this information in the report would be beneficial and quite reasonable replacement for RVTools

**Comment by tenthirtyam** [comment]
Sounds like an opportunity to extend the configuration report.
